modules/ROOT/pages/deployment-options.adoc

```diff
@@ -3,147 +3,133 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Anypoint Platform is an advanced enterprise platform for designing, developing, and managing APIs, 
-as well as for connecting APIs with applications, data, and devices. It enables you to deploy applications 
-anywhere on-premises, in the cloud, or in a hybrid environment to reliably address both small-scale and 
-mission-critical use cases.
+Anypoint Platform provides multiple deployment options, each of which differs based on data hosting requirements, the location and management of Anypoint Platform, and the individual Anypoint Platform components included: 
 
-Anypoint Platform functionality can be classified into two main categories:
+* US cloud
++
+The complete Anypoint Platform. All data is hosted within the US.
+* EU cloud
++
+Includes all US cloud components, but differs from the US cloud in that all data is hosted within the EU. Refer to xref:eu-control-plane::index.adoc[About the EU Control Plane] for additional information.
+* MuleSoft Government Cloud
++
+FedRAMP moderate authorized version of US cloud. Refer to xref:gov-cloud::index.adoc[About Government Cloud] for additional information.
+* Anypoint Platform PCE
++
+A single-tenant version of US cloud managed by your IT team. Anypoint Platform PCE supports on-premises and cloud installation and is specifically designed for customers with strict regulatory or compliance requirements that limit the use of cloud solutions. Refer to xref:private-cloud::index.adoc[About Anypoint Platform Private Cloud Edition] for additional information.  
 
-* xref:glossary#c[Control plane]
-+ 
-The Anypoint Platform control plane provides design, engagement, and management capabilities.
-* xref:glossary#r[Runtime plane] 
+== Anypoint Platform Location and Management
+Your organization's needs determine where Anypoint Platform functionality is deployed and who manages it. Anypoint Platform functionality can be classified into two main categories:
+
+* Control plane
 + 
-The Anypoint Platform runtime plane is where applications are deployed, and also where Mule runtime engine (Mule)
-and other application-related services, such as Anypoint Connectors, Anypoint Object Store, and messaging, run. 
+Anypoint Platform control plane functionality includes the design, engagement, and management capabilities of Anypoint Platform, such as Anypoint Design Center, Anypoint Management Center, and Anypoint Exchange.
+* Runtime plane
++
+Anypoint Platform runtime plane functionality includes the runtime engine (Mule) and runtime services, such as Anypoint Connectors, Anypoint Object Store, and messaging that allow you to scale the power of Mule and Anypoint Platform, and is where applications are deployed.
 
-The following table lists Anypoint Platform control and runtime plane components: 
+Anypoint Platform control plane and runtime plane functionality can be deployed on-premises, in the cloud, or in a hybrid environment in the following configurations:
```

Comment:
active voice?
```suggestion
You can deploy Anypoint Platform control plane and runtime plane functionality on-premises, in the cloud, or in a hybrid environment:
```
However, note that this intro list does not mimic the order in which the listed items are described, and it should.